cmd/activity-monitor/main.go

```diff
@@ -151,14 +146,12 @@ func main() {
 			syslogU, err := url.Parse(c.GlobalString("jsonlog"))
 			if err != nil {
 				log.Fatalf("Could not parse Syslog URL: %s", err)
-				return
 			}
 
 			logger.SetEndpoint(syslogU.Scheme, syslogU.Host)
 			err = logger.Connect()
 			if err != nil {
 				log.Fatalf("Could not open remote syslog: %s", err)
-				return
```

Comment:
I'm concerned by the removal of all these return statements.  The "log.Fatalf()" statements don't stop execution flow, so it seems like here and above, we would continue past the error condition.